src/Components/Workspace/PaperEdits/index.js

```diff
@@ -48,7 +49,8 @@ PaperEdits.propTypes = {
   handleDuplicateItem: PropTypes.any,
   handleEditItem: PropTypes.any,
   items: PropTypes.any,
-  projectId: PropTypes.any
+  projectId: PropTypes.any,
+  trackEvent: PropTypes.any
```

Comment:
Should this be 

```suggestion
  trackEvent: PropTypes.func
```
?

Reply:
good spot!